### PR TITLE
Update the UI of login page

### PR DESCRIPTION
**Pull request by nagiifan**
This PR fixes https://github.com/asyml/stave/issues/98. 

### Description of changes
Updating the UI of the login page by importing Material-UI components:
1. Adding logo 
2. Centering all text fields
3. Updating the input box and button UI 

### Possible influences of this PR
No negative influence expected

### Demonstration of results
Before: 
Screen Shot 2020-08-18 at 9 49 01 PM
After:
Screen Shot 2020-08-18 at 9 58 36 PM

### CODE REVIEW

**Comment by hunterhector** [comment]
You also needed to resolve the conflicts here:

image

**Comment by hunterhector** [comment]
I saw the "remember me" option at the login page. It seem it didn't work at the moment. What's the plan for this? One option is we comment out this for now until we implemented it.

Besides that, I have to search for the word "Bienvenue" myself. Any particular reasons for having that word in the signup page?

**Comment by hunterhector** [comment]
And we store the logos in the following path, so don't need to upload the logo again to the `src` root. In addition, it is advised to keep the `src` directory more organized and place resources into appropriate folders.

https://github.com/asyml/stave/tree/master/public

**Comment by nagiifan** [comment]
> I think the only thing lefts are:
> 
> 1. Comment out the "Remember me" for now since it does not function.
> 2. Remove the newly uploaded image and replace it with the one here "https://github.com/asyml/stave/tree/master/public"

All fixed.

File: src/app/pages/Login.tsx [hunk 1]
```diff
@@ -23,35 +29,81 @@ function Login() {
...
-          <input
-            placeholder="username"
-            onChange={e => setUsername(e.target.value)}
-            value={username}
-            name="username"
-          />
-        </div>
+      <Container component="main" maxWidth="xs">
+        <div className={classes.paper}>
+          <img alt='stave logo' src='https://github.com/asyml/stave/blob/master/public/Stave-dark-text@1x.png?raw=true'/>
```
Comment (hunterhector) [inline src/app/pages/Login.tsx:53]: The correct way to access the file to use the public URL like this:

https://create-react-app.dev/docs/using-the-public-folder/

The difference is, the project wouldn't rely on the service status of "Github". Instead, the logo file will be deployed to the user's server.

Comment (hunterhector) [reply src/app/pages/Login.tsx:53]: Or you can follow the following:

https://create-react-app.dev/docs/adding-images-fonts-and-files/

But using the github link is not recommended.

**Pull request merged**